server/routes/nl.py

```diff
@@ -54,8 +55,10 @@ def _get_preferred_type(types):
 
 
 def _maps_place(place_str):
+  if place_str.lower() in constants.SPECIAL_PLACE_REPLACEMENTS:
+    place_str = constants.SPECIAL_PLACE_REPLACEMENTS[place_str.lower()]
   api_key = current_app.config["MAPS_API_KEY"]
-  url_formatted = f"{MAPS_API}input={place_str}&key={api_key}"
+  url_formatted = f"{MAPS_API}query={place_str}&key={api_key}"
```

Comment:
To confirm, we're changing the param?  Add a comment / some reference link?

Reply:
Added a comment with reference and today's date.